### PR TITLE
Feature/uweb 6070 article author

### PR DESCRIPTION
**Pull request by zwaardje**

### CODE REVIEW

File: src/js/components/molecules/article-author/article-author.vue.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="article-author">
+    <c-image class="article-author__image" v-bind="image" />
```
Comment (rhelling) [inline src/js/components/molecules/article-author/article-author.vue.html]: image is optional ;-)

File: src/js/components/molecules/article-author/stories/article-author.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Article Author
+
+## DSM
+* [molecules/article-author](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/)
```
Comment (rhelling) [inline src/js/components/molecules/article-author/stories/article-author.md]: 💔

File: src/js/components/organisms/article-header/stories/article-header.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Article Header
+
+## DSM
+* [molecules/article-header](https://ultimaker.invisionapp.com/dsm/ultimaker/ultimaker-com/asset/components/)
```
Comment (rhelling) [inline src/js/components/organisms/article-header/stories/article-header.md]: 💔

File: src/js/components/molecules/article-author/stories/article-author.md (hunk 2 above)
Comment (zwaardje) [reply src/js/components/molecules/article-author/stories/article-author.md]: Not yet available

File: src/js/components/organisms/article-header/stories/article-header.md (hunk 3 above)
Comment (zwaardje) [reply src/js/components/organisms/article-header/stories/article-header.md]: Not yet available

**Pull request merged**